CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### Fixed
+
+- Fixed function and updated copy for "Go Back" buttons on map Idea Show page
```

Comment:
When updating the changelog, we need to keep GovSucces/BizDev in mind. They will need to understand what has changed. So here I would say something like "In a project with map view, you go back to the idea list instead of jumping to the top of the page when you click the go back button on an idea."

Reply:
That makes sense. I was trying to be concise like you might on a commit message, but I guess we don't have any space limitations in the changelog 👍 Will update it